moboff.py

```diff
@@ -159,10 +159,10 @@ def download(link, newdevice, video, delete,send):
     if send:
         for item in pb.chats:
             print(pb.chats.index(item), item, "\n")
```

Comment:
This can be made more pythonic.
E.g. : 

```
for i, device in enumerate(pb.devices, 1):
        print("{0} : {1}".format(i, device.nickname))
```